package.json

```diff
@@ -3,8 +3,12 @@
   "version": "3.0.2",
   "description": "webpack-web-app-manifest-plugin is a webpack plugin that generates a PWA manifest and integrates with the assets JSON generated by assets-webpack-plugin.",
   "main": "index.js",
+  "types": "types",
   "scripts": {
-    "test": "jest",
+    "build": "yarn tsc",
```

Comment:
do you need the `yarn` here?

Reply:
Not strictly necessary, no. [However, Yarn 2 encourages this](https://yarnpkg.com/getting-started/migration#call-binaries-using-yarn-run-rather-than-node_modulesbin).